webpack.prod.config.babel.js

```diff
@@ -128,19 +75,26 @@ const settings = {
     alias: {
       'normalize.css': 'normalize.css/normalize.css',
     },
-    root: [
+    modules: [
       path.resolve(__dirname),
       path.resolve('./src'),
+      'node_modules',
     ],
-    modulesDirectories: ['node_modules'],
-    extensions: ['', '.js', '.jsx'],
+    extensions: ['.js', '.jsx'],
   },
 };
 
 if (config.get('enablePostCssLoader')) {
-  settings.postcss = [
-    autoprefixer({ browsers: ['last 2 versions'] }),
-  ];
+  settings.plugins.push(
+    new webpack.LoaderOptionsPlugin({
+      options: {
+        context: path.resolve(__dirname),
+        postcss: [
+          autoprefixer({ browsers: ['last 2 versions'] }),
```

Comment:
One way it should be possible to know this is working is that if when it's disabled via conf the CSS should be smaller. That does assume there's something in the CSS that would need to be prefixed with though.